testing/trino-product-tests-launcher/src/main/resources/docker/trino-product-tests/conf/environment/multinode-apache-ranger/ranger-policymgr-ssl.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
```

Comment:
Is SSL/TLS security really configured and used? I see that you use HTTP for Ranger address.

Reply:
No, SSL /TLS is not covered. Covering SSL/TLS would require setting up a Ranger instance for the tests. As mentioned in a comment few months back, there is no published docker image for Apache Ranger. If such test coverage is a must-have, this PR will have to either wait until Ranger publishes a docker image or Trino tests should create a docker image by building Apache Ranger sources.